test/range/detail/random_access_iterator_test.cpp

```diff
@@ -223,6 +243,9 @@ TEST_F(random_access_iterator_test_fixture, equality)
     EXPECT_FALSE(it_v2 == it_w2);
     ++it_w2;
     EXPECT_TRUE(it_v2 == it_w2);
+    // const vs non-const
+    EXPECT_TRUE(it_v2 == it_w);
+    EXPECT_TRUE(it_w == it_v2);
```

Comment:
but only `==` not `!=` tested?

Reply:
well I also don't check <=

I thought that the operators themselves are checked elsewhere, this is only a check to see const VS non-const and assuming that the operators are implemented the same way (everything else would be ridiculous) this should be ok, or not?